### DISCUSSION TITLE
Adaya düşen korsanlar ve hindistan cevizleri

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#32] [Zeka Sorusu]
Issız bir adaya düşen 5 korsan, adada bol bulunan hindistan cevizlerinden bir yığın topluyorlar ve artık gece olduğundan paylaşma işini ertesi güne bırakıyorlar.

Gece uyanan 1.korsan nasıl olsa yarın paylaşacağız diye hindistan cevizi yığınını 5 eşit parçaya bölüyor, kendi payını alıyor artan bir hindistan cevizini de artık onların yanından hiç ayrılmayan maymuna veriyor.
2.uyanan da aynı işlemi tekrarlıyor 3. 4. ve 5. korsan da aynı işlemi tekrarlıyor. Ertesi sabah uyanıyorlar ve hiçbir şey olmamış gibi ortadaki yığını 5′e bölüp kalan 1 hindistan cevizini maymuna veriyorlar. Aslında hepside biliyor ki sabahki yığın akşamki yığın değil ama hepside aynı derecede suçlu olduğundan kimse bir şey söylemiyor.

Bu şartları sağlayan en küçük tamsayı nedir?

**Comment by yigidi** [comment]
Korsanların paylaşım işlemiyle ilgili verilen koşullara göre, asgari tamsayıyı bulmak için adımları sırasıyla inceleyebiliriz:

1. Korsanın işlemi: Yığındaki hindistancevizlerini 5 eşit parçaya böler, kendi payını alır ve artan bir hindistancevizini maymuna verir.
2. Korsanın işlemi: Gece boyunca olan biteni bilmeden yığındaki hindistancevizlerini 5 eşit parçaya böler, kendi payını alır ve artan bir hindistancevizini maymuna verir.
3. Korsanın işlemi: Aynı şekilde 5 eşit parçaya böler, kendi payını alır ve artan bir hindistancevizini maymuna verir.
4. Korsanın işlemi: Yine aynı şekilde 5 eşit parçaya böler, kendi payını alır ve artan bir hindistancevizini maymuna verir.
5. Korsanın işlemi: Son kez 5 eşit parçaya böler, kendi payını alır ve artan bir hindistancevizini maymuna verir.

Ertesi sabah, yığındaki hindistancevizleri tekrar 5 eşit parçaya bölerler ve kalan 1 hindistancevizini maymuna verirler.

Bu işlem sonunda, 5 korsanın her birinin gece boyunca aldığı hindistancevizlerin toplamı ile sabah aldıkları hindistancevizlerin toplamı aynı olmalıdır. Aksi takdirde, gece ve sabah yığınlardaki hindistancevizleri farklı olduğunu anlayabilirlerdi.

Bu koşulu sağlayan en küçük pozitif tamsayıyı bulmak için birkaç deneme yapabiliriz:

- 1 hindistancevizi vererek işlemi başlatırsak: 
  * 1. Korsan: 1 hindistancevizi alır, 0 artık hindistancevizi kalır.
  * 2. Korsan: 0 hindistancevizi alır, 0 artık hindistancevizi kalır.
  * 3. Korsan: 0 hindistancevizi alır, 0 artık hindistancevizi kalır.
  * 4. Korsan: 0 hindistancevizi alır, 0 artık hindistancevizi kalır.
  * 5. Korsan: 0 hindistancevizi alır, 0 artık hindistancevizi kalır.

Sabah: 0 hindistancevizi verilir. Gece ve sabah toplamları uyuşuyor.

En küçük pozitif tamsayı olan 1 bu koşulu sağlıyor. Bu nedenle, en küçük pozitif tamsayı 1'dir.

**Reply by sonsuzus** [reply]
Yığında 1 hindistan cevizi yoktur. Ayrıca denklemi sağlamıyor. Maymun bir kere 5 hindistan cevizi alıyor mutlaka.
